resources/benchmark-runner.mjs

```diff
@@ -663,3 +664,16 @@ export class SuiteRunner {
             await this._client.didRunTest(suite, test);
     }
 }
+
+// FIXME: implement async steps
+class AsyncSuiteRunner extends SuiteRunner {}
```

Comment:
I don't think it makes much sense to split the suite runner between sync & async. async runner is simply a superset of sync runner.